README.md

```diff
@@ -158,6 +158,7 @@ export SPRING_CLOUD_SERVICE=azure-spring-cloud-name # name of the service that w
 export LOG_ANALYTICS_WORKSPACE=log-analytics-name   # existing workspace or one that will be created in next steps
 export REGION=region-name                           # choose a region with Enterprise tier support
 ```
+Please visit [here](https://azure.microsoft.com/en-us/global-infrastructure/services/?products=spring-cloud&regions=all) for all available regions for Azure Spring Cloud.
```

Comment:
How about give an example value like "eastus" in this config and then following this   reference linking?